docs/infrastructure/eks-quickstart.md

```diff
@@ -2,6 +2,11 @@
 
 Follow the instructions below to set up an Amazon EKS cluster from scratch, including all the necessary underlying infrastructure, using Terraform.
 
+![codacy eks quicksart](images/codacy%20chart%20eks%20quickstart.png)
+
+The image above is a non-exhaustive overview of what you can expect to have
+deployed in your AWS account by using this quickstart.
+
```

Comment:
```suggestion
The diagram below presents a non-exhaustive architecture overview of what you can expect to have
deployed on your AWS account by following this quickstart tutorial.

![Codacy EKS quickstart architecture](images/codacy%20chart%20eks%20quickstart.png)
```

Reply:
It's better to rename the image file so it doesn't include spaces. For instance:

`eks-quickstart-architecture.png`

Reply:
Hi @paulopontesm, I noticed you did a force push to https://github.com/codacy/chart/commit/8b7ae40734b13a269fbeef54975bb35ea6923cce, but the commit is identical to the previous one. Was this intended?